### ISSUE TITLE
x/website: sidebar overlaps with footer on short screens

### ISSUE THREAD

**Issue by ankit-verma-209171** [golang/go#58134]
### What is the URL of the page with the issue?
 https://go.dev/solutions/clis
 
### What is your user agent?
 [Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36](https://developers.whatismybrowser.com/useragents/parse/?analyse-my-user-agent=yes)
 
 
### Screenshot
![image](https://user-images.githubusercontent.com/56779739/215321935-2e1bbcd9-c69d-4aaa-8e3a-345d8b2c56fe.png)
 
### What did you do?
 I was exploring use cases for the Go Language and the cli section on go.dev and found this issue

Steps to reproduce:
1. go to any specific use-case page on the website [eg. why go -> use cases -> clis](https://go.dev/solutions/clis)
2. scroll down to the bottom of the page
3. can see the overlapping aside section with footer

Can be reproducible for Chrome, and also on Edge browser 

### What did you expect to see?
content - aside section should not overlap with footer
 
 
### What did you see instead?
overlapping content - aside section with footer

![Screenshot (72)](https://user-images.githubusercontent.com/56779739/215324467-b51bb3d1-5e8c-442e-b0b5-c2fa010d59cf.png)

On inspecting the css, I found fixed position in the `.UseCaseSubNav-anchorLinks--sticky` class might be the cause
refer https://github.com/golang/website/blob/master/_content/css/styles.css#L3646-L3649

```css
.UseCaseSubNav-anchorLinks--sticky {
  position: fixed;
  top: 2rem;
}
```

### Possible Fix

While experimenting, I found that changing the `position` to `sticky` from `fixed` solves this issue
```css
.UseCaseSubNav-anchorLinks--sticky {
  position: sticky;
  top: 2rem;
}
```

 
![Screenshot (73)](https://user-images.githubusercontent.com/56779739/215324981-70249655-713b-4e50-bd07-563712697d12.png)

**Comment by mknyszek** [comment]
CC @golang/tools-team

**Comment by IanoNjuguna** [comment]
This is how it looks on my end
The client is: Linux Mint OS and Mozilla Firefox browser

![image](https://user-images.githubusercontent.com/52458657/215857423-9fd135c2-1577-4554-9084-46b28e737347.png)

**Comment by hyangah** [comment]
Thanks @ankit-verma-209171 for the detailed report.

cc @jamalc

**Comment by ankit-verma-209171** [comment]
I have attached a pull request (https://github.com/golang/website/pull/190) to this issue that resolves this issue. The pull request was moved to Gerrit (https://go-review.googlesource.com/c/website/+/464015) for the code review seven days ago still no review or comment; as it is my first time contributing to the Golang website, I need help and guidance on how to proceed further and would love feedback on the pull request.

I've test run the website with the changes locally using the codespace environment.
Thanks
